src/common/reducers/actions.js

```diff
@@ -274,8 +274,28 @@ export function downloadJava() {
       });
     });
 
-    await fse.copy(
-      path.join(javaBaseFolder, `${releaseName}-jre`),
+    // If NOT windows then tar.gz instead of zip, so we need to extract 2 times.
+    if (!(process.platform === "win32")) {
```

Comment:
This can be written as `if (process.platform !== "win32")`

Reply:
Ah right, was drawing a total blank on that.